### ISSUE TITLE
HTML5 Presentation multicolumn slide should be updated.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#509]
```
Please describe the issue:

With the release of Opera 11.10, the Opera browser now supports CSS3 
mulitcolumn with unprefixed properties.  The presentation styles should be 
updated to have "column-count", "column-rule", and "column-gap" in addition to 
the "-webkit-*" properties.

Please provide any additional information below.

http://dev.opera.com/articles/view/css3-multi-column-layout/
```

Original issue reported on code.google.com by `dominic...@inbox.com` on 20 Apr 2011 at 6:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 20 Apr 2011 at 6:34
- Changed state: **Assigned**
- Added labels: **Type-Feature**, **Milestone-6**
- Removed labels: **Type-Bug**

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 8 Dec 2011 at 12:29
- Added labels: **Milestone-Q42011-1**
- Removed labels: **Milestone-6**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 603bf1e5bbcd and Revision 3f0e09715ca1
```

Original comment by `han...@google.com` on 8 Dec 2011 at 4:21
- Changed state: **Fixed**
